ansible/slave.yml

```diff
@@ -5,17 +5,23 @@
   user: ubuntu
   vars:
    - jenkins_user: 'jenkins-build'
-   - exclusive: true
+   # jenkins API credentials:
+   - api_user: 'ceph-jenkins'
+   - token: '{{ token }}'
+   - api_uri: 'https://jenkins.ceph.com'
+   - nodename: '{{ nodename }}'
+   - labels: '{{ labels }}'
+
   tasks:
     - name: create a {{ jenkins_user }} user
-      user: name={{ jenkins_user }} comment="Jenkins Build Slave User" home="/home/{{ jenkins_user }}"
```

Comment:
I realized that you were merging the good parts that we already use in the .j2 file here